### PR TITLE
# Added project with github, github workflow and linters  

### PR DESCRIPTION
**Pull request by DyaryRaoof**
added linters like stylelint ,lighthouse and webhint to the repository.
added index.html to the repository.
added Hello Microverse h1 tag to the index.html file.
added style.css file
added a class of hello-microverse-header Hello Microverse h1 header and styled it with red color.
added .gitignore with the content of  node_modules/ and test.md
added test.md and added its name to .gitignore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+A new project to increatse our knowledge about Github , Github Flow and Pull requests.
```
Comment (emmanuellekamwa) [inline README.md]: Your README file is not descriptive enough. Please add some more details.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑🛠**
Hello @DyaryRaoof 👋🏽
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x] Pull Request should have a descriptive title telling you what changes are inside. Please try and fix that
- [x] Pull Request should have a short summary describing introduced changes in more detail. Please also try and fix that.

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (DyaryRaoof) [reply README.md]: Hi. Thank you for the review. I added the changes. could you look again?

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+A new project to increase our knowledge about Github , Github Flow and Pull requests. we used linters like lighthouse , webhint , and stylelint. it also contains basic html and css files with basic content like head and body tags as well as styled h1 elements in the header tag.
```
Comment (icebox827) [inline README.md]: - [ ] **Required:** Please use the template provide by Microverse to create a descriptive README file. Click on the following link for more information [README](https://github.com/microverseinc/readme-template)

**Review by icebox827 — CHANGES_REQUESTED** [review]
Hi Dyary, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] The PR should have a descriptive title. Use '# ' as Markdown at the beginning of the sentence to turn it into a Title
_Check the comments under the review._



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @DyaryRaoof  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**